packages/components/src/range-control/README.md

```diff
@@ -294,6 +294,15 @@ Define if separator line under/above control row should be disabled or full widt
 - Required: No
 - Platform: Mobile
 
+#### type
+
+Define if the value selection should present a stepper control or a slider control in the bottom sheet on mobile. Defaults to `slider` if no option is provided.
+
+- Type: `String Enum`
+- Values: `none` | `slider` | `stepper`
+- Required: No
+- Platform: Mobile
```

Comment:
I wonder about these values since the only value that will be checked is `stepper` so maybe we could remove `slider` as an option?

What do think about something like:

```suggestion
Define if the value selection should present a stepper control or a slider control in the bottom sheet on mobile. To use the stepper set the type value as  `stepper`. Defaults to slider if no option is provided.

- Type: `String`
- Required: No
- Platform: Mobile
```

Reply:
This works for me!